web/themes/custom/pineapple/assets/js/menu.behavior.js

```diff
@@ -0,0 +1,13 @@
+(function ($, Drupal, drupalSettings) {
+
+  'use strict';
+
+  Drupal.behaviors.menu = {
+    attach: function (context, settings) {
```

Comment:
Add js to remove responsive class on large screens (> 768), here is an example of code and library used.  https://gist.github.com/ElXyfer/4024f83b5e47241e00efbed9fa55e97e

Reply:
done